queue/queue.py

```diff
@@ -1,15 +1,18 @@
 class Queue:
```

Comment:
How could this be implemented with a Linked List? Not required to change it, but this is why `self.size` is provided so that you can increment / decrement the size when adding or removing nodes from the linked list

Reply:
Provided pseudocode answers here: https://github.com/rd825/Data-Structures/pull/1/commits/0b6d988842adb428cb4b6dd7b4546304a204bf3d

Reply:
Thanks for adding that. Looks good